cognite/client/_cognite_client.py

```diff
@@ -75,6 +76,7 @@ def __init__(self, config: ClientConfig | None = None) -> None:
         self.functions = FunctionsAPI(self._config, self._API_VERSION, self)
         self.data_modeling = DataModelingAPI(self._config, self._API_VERSION, self)
         self.documents = DocumentsAPI(self._config, self._API_VERSION, self)
+        self.workflows = WorkflowAPI(self._config, self._API_VERSION, self)
```

Comment:
Consider just passing `"beta"` here? Maybe we should do this for other APIs that are "fully in beta (or alpha)"? @erlendvollset wdyt?
```suggestion
        self.workflows = WorkflowAPI(self._config, "beta", self)
```